diff --git a/contrib/plot/workrave-dump b/contrib/plot/workrave-dump
old mode 100644
new mode 100755
index c432bcd..f721b33
--- a/contrib/plot/workrave-dump
+++ b/contrib/plot/workrave-dump
@@ -1,6 +1,8 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl
 
 use strict;
+use warnings;
+use warnings::register;
 use integer;
 use POSIX qw(strftime);
 use Time::Local;
@@ -51,13 +53,19 @@ sub dumpdata($$$$$)
 {
 	my ($tmdate, $break, $misc, $breakstats, $miscstats) = @_;
 
-	die "No timestamp at line $.\n" if !defined($tmdate);
-	die "No breaks at line $.\n" if !defined($break);
+	warnings::warn "No timestamp at line $.\n" if !defined($tmdate);
+	warnings::warn "No breaks at line $.\n" if !defined($break);
+
 	foreach my $breaktype (0..2) {
-		die "No breaktype #$breaktype at line $.\n"
-			if !defined($break->{$breaktype});
+		if (!defined($break->{$breaktype})) {
+			warnings::warn "No breaktype #$breaktype at line $.\n";
+			return;
+		}
+	}
+	warnings::warn "No misc at line $.\n" if !defined($misc);
+	if (!defined($tmdate) || !defined($break) || !defined($misc)) {
+		return;
 	}
-	die "No misc at line $.\n" if !defined($misc);
 
 	my $ts = strftime("%Y%m%d%H%M", gmtime(timegm(0,
 		@$tmdate{qw(min hour mday mon year)})));
@@ -85,7 +93,7 @@ my $step = 24 * 3600;	# 24-hr steps
 my $heartbeat = 4 * $step;
 
 my (@DS);
-my @breaktypes qw(micro rest daily);
+my @breaktypes = qw(micro rest daily);
 foreach my $breaktype (@breaktypes) {
 	foreach my $breakstat (@breakstats) {
 		my $key = "$breaktype$breakstat";
@@ -106,6 +114,7 @@ my ($tmdate, $break, $misc) = (undef, undef, undef);
 open(SOURCE, $source) or die "open $source: $!\n";
 while (<SOURCE>) {
 	chomp;
+	s/^\0*//; # remove corrupted data
 	if (/^WorkRaveStats 4$/) {
 		# Ignore header
 		next;
