--- backend/src/osx/OSXConfigurator.cc-orig	2009-09-21 09:39:25.000000000 -0700
+++ backend/src/osx/OSXConfigurator.cc	2009-09-21 09:07:43.000000000 -0700
@@ -116,6 +116,7 @@
           // FALLTHROUGH
 
         case VARIANT_TYPE_STRING:
+        {
           NSString *val = [[NSUserDefaults standardUserDefaults] stringForKey: keystring];
           if (val != nil)
             {
@@ -126,6 +127,7 @@
               ret = false;
             }
           break;
+        }
 
         default:
           ret = false;
