---
 frontend/gtkmm/src/GUI.cc |    3 ---
 1 file changed, 3 deletions(-)

Index: workrave-1.9.0/frontend/gtkmm/src/GUI.cc
===================================================================
--- workrave-1.9.0.orig/frontend/gtkmm/src/GUI.cc	2009-09-09 18:18:42.000000000 -0500
+++ workrave-1.9.0/frontend/gtkmm/src/GUI.cc	2009-09-09 18:24:44.394028585 -0500
@@ -198,6 +198,12 @@
   
   Gtk::Main kit(argc, argv);
 
+  if (!g_thread_supported())
+    {
+      g_thread_init(NULL);
+    }
+
+
   init_core();
   init_nls();
   init_platform();
@@ -328,14 +334,6 @@
   [ [ AppController alloc ] init ];
 #endif
   
-#if defined (PLATFORM_OS_WIN32) || defined(PLATFORM_OS_OSX)
-  // Win32/OSX need this....
-  if (!g_thread_supported())
-    {
-      g_thread_init(NULL);
-    }
-#endif
-
 #if defined(PLATFORM_OS_UNIX)
   char *display = gdk_get_display();
   System::init(display);
