diff -ru workrave-1.8.3.old/frontend/common/src/gnome/GnomeSoundPlayer.cc workrave-1.8.3/frontend/common/src/gnome/GnomeSoundPlayer.cc
--- workrave-1.8.3.old/frontend/common/src/gnome/GnomeSoundPlayer.cc	2004-02-29 15:24:21.000000000 +0100
+++ workrave-1.8.3/frontend/common/src/gnome/GnomeSoundPlayer.cc	2006-08-06 22:18:35.000000000 +0200
@@ -1,6 +1,6 @@
 // GnomeSoundPlayer.cc --- Sound player
 //
-// Copyright (C) 2002, 2003, 2004 Rob Caelers & Raymond Penners
+// Copyright (C) 2002, 2003, 2004, 2006 Rob Caelers & Raymond Penners
 // All rights reserved.
 //
 // This program is free software; you can redistribute it and/or modify
@@ -27,9 +27,21 @@
 #include <debug.hh>
 
 #include <libgnome/gnome-triggers.h>
+#include <libgnome/gnome-sound.h>
 
 GnomeSoundPlayer::GnomeSoundPlayer()
 {
+  gnome_sound_init (NULL);
+  gnome_sound_sample_load ("workrave/exercise_ended",     DATADIR "/sounds/workrave/exercise-ended.wav");
+  gnome_sound_sample_load ("workrave/break_ignored",      DATADIR "/sounds/workrave/break-ignored.wav");
+  gnome_sound_sample_load ("workrave/break_prelude",      DATADIR "/sounds/workrave/break-prelude.wav");
+  gnome_sound_sample_load ("workrave/daily_limit",        DATADIR "/sounds/workrave/daily-limit.wav");
+  gnome_sound_sample_load ("workrave/exercise_ended",     DATADIR "/sounds/workrave/exercise-ended.wav");
+  gnome_sound_sample_load ("workrave/exercises_ended",    DATADIR "/sounds/workrave/exercises-ended.wav");
+  gnome_sound_sample_load ("workrave/micro_break_ended,", DATADIR "/sounds/workrave/micro-break-ended.wav");
+  gnome_sound_sample_load ("workrave/micro_break_started",DATADIR "/sounds/workrave/micro-break-started.wav");
+  gnome_sound_sample_load ("workrave/rest_break_ended",   DATADIR "/sounds/workrave/rest-break-ended.wav");
+  gnome_sound_sample_load ("workrave/rest_break_started", DATADIR "/sounds/workrave/rest-break-started.wav");
 }
 
 GnomeSoundPlayer::~GnomeSoundPlayer()
diff -ru workrave-1.8.3.old/frontend/common/src/gnome/Makefile.am workrave-1.8.3/frontend/common/src/gnome/Makefile.am
--- workrave-1.8.3.old/frontend/common/src/gnome/Makefile.am	2006-01-22 17:44:28.000000000 +0100
+++ workrave-1.8.3/frontend/common/src/gnome/Makefile.am	2006-08-06 22:19:52.000000000 +0200
@@ -1,6 +1,6 @@
 # Process this file with automake to produce Makefile.in
 #
-# Copyright (C) 2002, 2003 Rob Caelers & Raymond Penners
+# Copyright (C) 2002, 2003, 2006 Rob Caelers & Raymond Penners
 #
 # $Id: Makefile.am,v 1.2 2006/01/22 16:44:28 rcaelers Exp $
 #
@@ -20,6 +20,7 @@
 
 libworkrave_frontend_common_gnome_a_CXXFLAGS = \
 			-DWORKRAVE_DATADIR="\"${pkgdatadir}\"" \
+			-DDATADIR="\"${datadir}\"" \
 			@X_CFLAGS@ \
 			-D_XOPEN_SOURCE=600 -W -I.. -I../../include \
 			@WR_COMMON_CFLAGS@ @WR_BACKEND_INCLUDES@ \
diff -ru workrave-1.8.3.old/frontend/common/src/gnome/Makefile.in workrave-1.8.3/frontend/common/src/gnome/Makefile.in
--- workrave-1.8.3.old/frontend/common/src/gnome/Makefile.in	2006-05-13 00:19:19.000000000 +0200
+++ workrave-1.8.3/frontend/common/src/gnome/Makefile.in	2006-08-06 22:21:39.000000000 +0200
@@ -432,6 +432,7 @@
 
 @HAVE_GNOME_TRUE@libworkrave_frontend_common_gnome_a_CXXFLAGS = \
 @HAVE_GNOME_TRUE@			-DWORKRAVE_DATADIR="\"${pkgdatadir}\"" \
+@HAVE_GNOME_TRUE@			-DDATADIR="\"${datadir}\"" \
 @HAVE_GNOME_TRUE@			@X_CFLAGS@ \
 @HAVE_GNOME_TRUE@			-D_XOPEN_SOURCE=600 -W -I.. -I../../include \
 @HAVE_GNOME_TRUE@			@WR_COMMON_CFLAGS@ @WR_BACKEND_INCLUDES@ \
